config/default/system.site.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Archaeology Center'
```

Comment:
This one will definitely have to be split out to each site.

Reply:
yeah. i'll set the `default` to something and add the override